src/AggregateNugetPackages/Microsoft.Net.RoslynDiagnostics/NuGet/Microsoft.Net.RoslynDiagnostics.props

```diff
@@ -0,0 +1,62 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition="'$(ProjectLanguage)' == 'CSharp'">
+    <Analyzer Include="$(NuGetPackageRoot)\Microsoft.ApiDesignGuidelines.Analyzers\$(RoslynDiagnosticsNugetPackageVersion)\analyzers\dotnet\cs\Analyzer.Utilities.dll" />
```

Comment:
@lgolding We will need to add entries for C# and VB specific utility assemblies here once you have merged your change which adds these assemblies.